### ISSUE TITLE
[Docs] Create video/interactive versions of the new Redux tutorials

### ISSUE THREAD

**Issue by markerikson** [reduxjs/redux#4024]
Last year I rewrote the Redux core docs tutorials from scratch. We now have the "Essentials" tutorial, which teaches RTK as the default syntax for writing Redux logic, and the "Fundamentals" tutorial, which teaches the underlying principles and mechanics and finishes with showing how RTK simplifies your Redux code.

However, both tutorials are very long (25K words), and are in written form.  It's clear that many people prefer learning from videos.

Dan Abramov's Egghead videos on Redux are still amazing for learning how Redux works from the bottom up, but the approaches they show are unfortunately outdated at this point: separate actions/constants/reducers, `SCREAMING_SNAKE_CASE` variables, etc.

I would love to see someone else adapt the Essentials and Fundamentals written tutorials into a video course format, or an interactive Scrimba-style course like the Vue docs have, and then we could link those in the docs as additional learning options.

**Comment by xjamundx** [comment]
I'm actively working on an RTK course for egghead, but it's not based off of your essentials material. Happy to provide coaching or support if you wanted to do one. Or after this one is live (about a month out) would be more than willing to work on a new one based on official redux material that you could even post on the website or whatever.

**Comment by markerikson** [comment]
@xjamundx yeah, would definitely love to collaborate with you in some way!  I can see value in multiple potential courses, really:

- You already did one that's "migrating `connect` to hooks"
- One focused on RTK specifically would be useful (APIs, usage patterns, adoption)
- So would versions of the "Essentials" and "Fundamentals" tutorials specifically
- And RTKQ easily merits its own entire course, and I know @phryneas has said he might be interested in that (and who better?)

**Comment by phryneas** [comment]
@xjamundx  Yup, if something comes up there I'd be in for that. 
I [did some videos](https://www.youtube.com/watch?v=FDEgXmx1V4A) when we first released the RTKQ alpha and it was a rough experience, so "coaching" sounds like a great learning opportunity!

**Comment by xjamundx** [comment]
I have two right now:
https://egghead.io/courses/modernizing-a-legacy-redux-application-with-react-hooks-c528
https://egghead.io/courses/redux-with-react-hooks-8a37

And then the new one is going to be called :
**Build an Online Store with React, React Router, RTK, CSS Modules, Vite and TypeScript**

Which covers a lot, but it's kind of nice to build an entire application from scratch and seeing how all the piece fit together and how each part builds on the other. The store part is kind of fake, but a shopping cart where every page can see the current number of items in the count I thought was a good way to demonstrate selectors, memoized selectors and redux in general. I also focus on adding Types and Tests for all the redux-related code, which has been a bit of a blind spot for some folks.

Let me get this one finished and then happy to collab!

**Comment by markerikson** [comment]
It's several years later. I ain't got the time :)
